go/common/version/version.go

```diff
@@ -127,3 +128,26 @@ func parseSemVerStr(s string) Version {
 
 	return Version{Major: semVers[0], Minor: semVers[1], Patch: semVers[2]}
 }
+
+// Convert Go Modules compatible version to Oasis Core's canonical version.
+func ConvertGoModulesVersion(goModVersion string) string {
+	re := regexp.MustCompile(`v0.(?P<year>[0-9]{2})(?P<minor>[0-9]{2}).(?P<micro>[0-9]+)`)
+	match := re.FindStringSubmatch(goModVersion)
+	// NOTE: match[0] contains the whole matched string.
+	if len(match) != 4 {
+		return "undefined"
```

Comment:
Move this to a constant (e.g., `VersionUndefined` or something).

Reply:
Done.